### ISSUE TITLE
Mint 0.10.0 causes carthage to fail

### ISSUE THREAD

**Issue by mpodeszwa** [yonaskolb/Mint#80]
After upgrading mint from 0.9.0 to 0.10.0 I can't build my xcode project anymore.
This is the command I run in xcode build phase:
```
mint run carthage carthage copy-frameworks
```
And output:
```
🌱  Using "Carthage/Carthage" "0.29.0" from Mintfile.
🌱  carthage 0.29.0 already installed
🌱  Running carthage 0.29.0...
Environment variable not set: SCRIPT_INPUT_FILE_COUNT
Command /bin/sh failed with exit code 1
```

Command `carthage copy-frameworks` without mint works fine.

**Comment by yonaskolb** [comment]
Fixed and released in 0.10.1

**Comment by mpodeszwa** [comment]
Thanks @yonaskolb , works well now 👍
